rules/linux/execution_shell_evasion_linux_binary.toml

```diff
@@ -0,0 +1,104 @@
+[metadata]
+creation_date = "2022/05/06"
+maturity = "production"
+updated_date = "2022/05/09"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary(s) abuse to breakout out of restricted shells or environments by spawning an interactive system
+shell. The linux utility(s) activity of spawning shell is not a standard use of the binary for a user or system
+administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their
+access.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via  Linux Binary(s)"
+references = [
+    "https://gtfobins.github.io/gtfobins/apt/",
+    "https://gtfobins.github.io/gtfobins/apt-get/",
+    "https://gtfobins.github.io/gtfobins/nawk/",
+    "https://gtfobins.github.io/gtfobins/mawk/",
+    "https://gtfobins.github.io/gtfobins/awk/",
+    "https://gtfobins.github.io/gtfobins/gawk/",
+    "https://gtfobins.github.io/gtfobins/busybox/",
+    "https://gtfobins.github.io/gtfobins/c89/",
+    "https://gtfobins.github.io/gtfobins/c99/",
+    "https://gtfobins.github.io/gtfobins/cpulimit/",
+    "https://gtfobins.github.io/gtfobins/crash/",
+    "https://gtfobins.github.io/gtfobins/env/",
+    "https://gtfobins.github.io/gtfobins/expect/",
+    "https://gtfobins.github.io/gtfobins/find/",
+    "https://gtfobins.github.io/gtfobins/flock/",
+    "https://gtfobins.github.io/gtfobins/gcc/",
+    "https://gtfobins.github.io/gtfobins/mysql/",
+    "https://gtfobins.github.io/gtfobins/nice/",
+    "https://gtfobins.github.io/gtfobins/ssh/",
+    "https://gtfobins.github.io/gtfobins/vi/",
+    "https://gtfobins.github.io/gtfobins/vim/",
+    "https://gtfobins.github.io/gtfobins/capsh/",
+    "https://gtfobins.github.io/gtfobins/byebug/",
+]
+risk_score = 47
+rule_id = "52376a86-ee86-4967-97ae-1a05f55816f0"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+
+  /* launch shells from unusual process */
+  (process.name == "capsh" and process.args == "--") or
+
+  /* launching shells from unusual parents or parent+arg combos */
+  (process.name in ("bash", "sh", "dash","ash") and
+    (process.parent.name == "byebug") or
+    (process.parent.name == "crash" and process.parent.args == "-h") or
```

Comment:
This depends on pager which is covered elsewhere in the rule (and also would not be a child of the shell)
```suggestion
```